addons/point_of_sale/static/src/app/screens/payment_screen/payment_lines/payment_lines.xml

```diff
@@ -19,14 +19,10 @@
                                 </div>
                             </div>
                             <t t-if="!line.payment_status or !['done', 'reversed', 'waitingCard', 'waitingCapture'].includes(line.payment_status)">
-                                <button t-if="ui.isSmall" class="btn btn-link mx-2 px-3"
-                                aria-label="Edit" title="Edit" t-on-click="() => this.selectLine(line)">
-                                    <i class="fa fa-2x fa-edit" />
-                                </button>
```

Comment:
why is this removed?

Reply:
FP requested to remove this icon/btn-link. You can still edit by clicking on the line afair.